projects/ngx-xapi/README.md

```diff
@@ -1,24 +1,183 @@
-# NgxXapi
+# ngx-xapi
 
-This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 16.1.0.
+Lightweight Angular wrapper for the
+[xAPI](https://xapi.com).
 
-## Code scaffolding
+It can be used to connect any xAPI compitable LRS (learning record store).
```

Comment:
check for spelling mistakes 

e.g. compitable

Copilot will help you and you should add [code spell checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) to VS Code.